bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieShellTest.java

```diff
@@ -408,5 +409,21 @@ public void testListBookiesCmdReadWrite() throws Exception {
             .apply(same(shell.bkConf), same(mockListBookiesFlags));
         verify(mockListBookiesFlags, times(1)).readonly(false);
         verify(mockListBookiesFlags, times(1)).readwrite(true);
+        verify(mockListBookiesFlags, times(1)).all(false);
```

Comment:
Is this test passing? I would expect 'false' here

Reply:
I think that if you pass "listbookies", "-rw"  the result is the available read/write bookies
If you pass "listbookies", "-a" you see all the bookies (including the readonly/unavailable)